Microsoft.DotNet.Interactive.Tests/LanguageKernelTests.cs

```diff
@@ -959,40 +960,33 @@ public async Task When_SubmitCode_command_adds_packages_to_kernel_then_PackageAd
 
             events
                 .Should()
-                .Contain(e => e is DisplayedValueProduced && ((DisplayedValueProduced)e).Value.ToString().Contains("Installing"));
-
-            events
-                .Should()
-                .Contain(e => e is DisplayedValueUpdated && ((DisplayedValueUpdated)e).Value.ToString().Contains("done!"));
+                .ContainSingle(e => e is DisplayedValueProduced && ((DisplayedValueProduced)e).Value.ToString().Contains("Installing package"));
 
             events
                 .Should()
-                .ContainSingle(e => e is NuGetPackageAdded);
+                .ContainSingle(e => e is DisplayedValueUpdated && ((DisplayedValueUpdated)e).Value.ToString().Contains("done!"));
```

Comment:
This looks like a much weaker assertion than checking for the specific event type.

Reply:
Immediately below is this assertion, which accurately verifies the event:
This line verifies that the updated output text is produced.

````
            events.OfType<PackageAdded>()
                  .Should()
                  .BeEquivalentTo(new NugetPackageReference("Microsoft.Extensions.Logging", "2.2.0"));
                  .ContainSingle(e => ((PackageAdded)e).PackageReference.PackageName == "Microsoft.Extensions.Logging" 
                                   && ((PackageAdded)e).PackageReference.PackageVersion == "2.2.0");

````